pkg/audit/newverifier_test.go

```diff
@@ -20,6 +20,7 @@ func TestVerifierHappyPath(t *testing.T) {
 	testplanet.Run(t, testplanet.Config{
 		SatelliteCount: 1, StorageNodeCount: 6, UplinkCount: 1,
 	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		t.Skip("flaky")
```

Comment:
do you think it would be helpful to add a `TODO` comment about the details you mentioned in slack enginnering chat?.... i.e. "the deal is, NextStripe needs to be refactored (the test fail is like a symptom of a larger problem), but we don’t have a plan to refactor it this second."  https://storj.slack.com/archives/CAQV0AE2Z/p1554224990378200

Reply:
I'll add it if you think it's helpful! 👍

Reply:
yeah i think its important to know why we are skipping and how it will be fixed moving forward.